### PR TITLE
added footnotes (twitter ,facebook,about us)

### LINKED ISSUE #3: Add Some Footer Content
Please add some footer content in homepage
Footer content
     1. Facebook
      2. Twitter
      3. About

Capture

@rajat2502 Please review the issue

### PR DESCRIPTION
**Pull request by pratik0316**
added the footnotes

Fixes #3 
IMG_20191201_124036

### CODE REVIEW

**Comment by rajat2502** [comment]
@Pratikrocks Fix the footer to the bottom of the viewport when the content doesn't take the full height of the page and to the end of the content when there is scrollable content. 
Also, style the footer properly it doesn't good right now. 
The footer content should be something like **"Made by these talented Persons"** where '**there**' is a link to the list of contributors.

**Comment by sgpritam** [comment]
Add more option in footer facebook ,twitter,Instagram,gmail bla bla...
https://github.com/NJACKWinterOfCode/Ravenous/pull/7#issuecomment-560073973

File: src/components/Business/Business.js [hunk 1]
```diff
@@ -1,6 +1,8 @@
 import React from 'react';
 import './Business.css';
 
+
+
```
Comment (rajat2502) [inline src/components/Business/Business.js]: Remove extra spaces like this from the file

File: src/components/BusinessList/BusinessList.js [hunk 2]
```diff
@@ -15,13 +15,36 @@ class BusinessList extends React.Component {
...
+        <a href="#"><img src ={require('./twitter-icon.jpg')} alt="Twitter"></img></a>
+      </div>
+      <div className="col">
+        <a href="#"><img src ={require('./gmail-icon.png')} alt="Twitter" height="80" width="80" ></img></a>
+      </div>
+      <div className="col">
+        <a href="#"><img src ={require('./instagram-icon.jpeg')} alt="Twitter" height="80" width="80"></img></a>
+      </div>
+      
+
```
Comment (rajat2502) [inline src/components/BusinessList/BusinessList.js]: Remove these extra lines

**Review by rajat2502 — CHANGES_REQUESTED** [review]

File: src/components/Business/Business.js (hunk 1 above)
Comment (pratik0316) [reply src/components/Business/Business.js]: @rajat2502 done

File: src/components/BusinessList/BusinessList.js (hunk 2 above)
Comment (pratik0316) [reply src/components/BusinessList/BusinessList.js]: @rajat2502  done

**Comment by pratik0316** [comment]
Facebook ,Instagram,AboutUs,Twitter icons are apperaing at the bottom of the page

Screenshot from 2019-12-07 16-23-39
Screenshot from 2019-12-07 16-23-57
Screenshot from 2019-12-07 16-24-00

**Comment by rajat2502** [comment]
It doesn't look good to me!!!

**Pull request closed without merging**